plutus-conformance/tplc/typecheck/Spec.hs

```diff
@@ -1,15 +0,0 @@
-{-# LANGUAGE OverloadedStrings #-}
-module Main (main) where
```

Comment:
Gone because it wasn't doing anything?

Reply:
Those were skeleton files in another PR showing the future structure of the test suites, but we've decided to change the structure since.